src/utilities.js

```diff
@@ -1,13 +1,26 @@
 export const fetchUserLocation = () => {
   return fetch('http://api.airvisual.com/v2/nearest_city?key=26e9573a-6960-4337-b548-ec068499ad9f')
+    .then(response => handleFetchErrors(response))
```

Comment:
Could we put this call of `handleFetchErrors()` inside the same .then as the `response.json()` ?  that might clean this up nicely